### PR TITLE
Save change features as GeoJson

### PR DESCRIPTION
**Pull request by matthieun**
### Description:

Save the multiple stages of change objects as geojson.
- Save FeatureChange objects as GeoJson Features
- Save Change objects as GeoJson FeatureCollections

### Potential Impact:

New feature

### Unit Test Approach:

Added tests for each feature type.

### Test Results:

Tests pass

------

In doubt: [Contributing Guidelines](CONTRIBUTING.md)

### CODE REVIEW

**Comment by lucaspcram** [comment]
Let's take a look at `serializedAreaFull.json` (one of the test resources you added), and see how we could convert it to GeoJSON. Currently, it looks like:

```
{
  "type": "ADD",
  "boundsWKT": "POLYGON ((-122.0314121 37.3901066, -122.0314121 37.3909505, -122.0306289 37.3909505, -122.0306289 37.3901066, -122.0314121 37.3901066))",
  "atlasEntityReference": {
    "type": "AREA",
    "class": "org.openstreetmap.atlas.geography.atlas.bloated.BloatedArea",
    "identifier": 123,
    "tags": "{tagKey1=tagValue1, tagKey2=tagValue2}",
    "relations": "[555, 444]",
    "WKT": "POLYGON ((-122.0310473 37.3909505, -122.0314121 37.3903197, -122.0311332 37.3901066, -122.0306289 37.3908482, -122.0310473 37.3909505, -122.0310473 37.3909505))",
    "GEOJSON": {
      "type": "Polygon",
      "coordinates": [
        [
          [
            -122.0310473,
            37.3909505
          ],
          [
            -122.0314121,
            37.3903197
          ],
          [
            -122.0311332,
            37.3901066
          ],
          [
            -122.0306289,
            37.3908482
          ],
          [
            -122.0310473,
            37.3909505
          ]
        ]
      ]
    }
  },
  "boundsGEOJSON": {
    "type": "Polygon",
    "coordinates": [
      [
        [
          -122.0314121,
          37.3901066
        ],
        [
          -122.0314121,
          37.3909505
        ],
        [
          -122.0306289,
          37.3909505
        ],
        [
          -122.0306289,
          37.3901066
        ],
        [
          -122.0314121,
          37.3901066
        ]
      ]
    ]
  }
}
```
With just a few minor tweaks, we could change it to be fully compliant with [the GeoJSON spec.](https://tools.ietf.org/html/rfc7946) Notice the use of [GeoJSON's special "bbox" property](https://tools.ietf.org/html/rfc7946#section-5), which handles our `Located` use case nicely.
```
{
       "type": "Feature",
       "bbox": [-122.0314121, 37.3901066, -122.0306289, 37.3909505],
       "geometry": {
           "type": "Polygon",
           "coordinates": [
                 [
                       [-122.0310473, 37.3909505],
                       [-122.0314121, 37.3903197],
                       [-122.0311332, 37.3901066],
                       [-122.0306289, 37.3908482],
                       [-122.0310473, 37.3909505]
                 ]
            ]
       },
       "properties": {
               "featureChangeType": "ADD",
               "entityType": "AREA",
               "class": "org.openstreetmap.atlas.geography.atlas.bloated.BloatedArea",
               "identifier": 123,
               "tags": "{tagKey1=tagValue1, tagKey2=tagValue2}",
               "relations": "[555, 444]",
               "WKT": "... omitted for brevity ...",
               "boundsWKT": "... omitted for brevity ..."
       }
   }
```
The spec is pretty extensible as well, [see this section on foreign members.](https://tools.ietf.org/html/rfc7946#section-6). Another advantage to using compliant GeoJSON from the get-go is that we get full compatibility with existing visualization tools (like QGIS or [geojson.io](http://geojson.io/#map=2/20.0/0.0) or whatever). We could then serialize an entire `Change` object [using a feature collection,](https://tools.ietf.org/html/rfc7946#section-3.3) which would make `Change`s easily visualizable.

**Comment by matthieun** [comment]
That is a good idea actually, let me do that!

File: src/main/java/org/openstreetmap/atlas/geography/Rectangle.java [hunk 1]
```diff
@@ -224,6 +226,16 @@ public Envelope asEnvelope()
                 this.upperRight.getLatitude().asDegrees());
     }
 
+    public JsonArray asGeoJsonBbox()
```
Comment (lucaspcram) [inline src/main/java/org/openstreetmap/atlas/geography/Rectangle.java]: 👍

**Comment by matthieun** [comment]
@lucaspcram ready!

**Pull request merged**